readme.md

```diff
@@ -18,3 +18,21 @@ A simple executable that renames image files based on keywords that describe the
 Navigate to a folder with some images in it and run `airenamer`.
 
 Make sure you have a backup first. It only copies the files, but I'm not responsible for any data loss.
+This is useful when you have Ollama running on remote host and you want to use different model.
```

Comment:
what is 'this'. not sure how it relates to the previous sentence.

Reply:
This was a typo and now it is corrected in latest update.